### PR TITLE
Add nestandard1.7 build for ResourceManager

### PR DESCRIPTION
**Pull request by ericstj**
This retargets the ref and src to netstandard1.7. It splits up the test
to run against both netstandard1.3 and netstandard1.7.

/cc @alexperovich @weshaggard @danmosemsft @stephentoub

### CODE REVIEW

**Comment by ericstj** [comment]
@alexperovich I know you were working on some API additions so feel free to just grab this instead of merging it separately.  I can close the PR if that's what you prefer.

File: src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <AssemblyName>System.Resources.ResourceManager.Tests</AssemblyName>
     <RootNamespace>System.Resources.Tests</RootNamespace>
     <ProjectGuid>{1D51A16C-B6D8-4E8F-98DE-21AD9A7062A1}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NugetTargetMoniker>
```
Comment (alexperovich) [inline src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj]: Should this be netstandard1.7?

**Comment by weshaggard** [comment]
cc @kouvel as I think I saw you doing similar changes in your PR.

**Comment by ericstj** [comment]
I'm fine closing this one, just wanted to get the example up there so that folks working on the library weren't blocked.

**Pull request closed without merging**